spec/system/admin_variant_retirement_create_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe 'admin can retire a variant for a split' do
   let(:split_page) { app.admin_split_show_page }
-  let(:variant_page) { app.admin_variant_details_edit_page }
+  let!(:variant_page) { app.admin_variant_details_edit_page }
```

Comment:
part of a dummy commit, since actions failed for the second to last commit due to docker

Reply:
If you do `git commit --amend` (to rewrite the last commit) and then `get push -f` (to force push to the branch) I think you can accomplish something similar. But GH actions should also be re-runnable through the GH UI.

Reply:
duh, didn't think of using `--amend` 🙃 (I did think of retrying from the UI, but couldn't find the button to do so? and figured it's maybe because I forked this repo (but it could just be me missing it))

Reply:
Ok, undid this (and had the chance to try out `--amend` in the process)